### PR TITLE
Send Chat Message endpoint

### LINKED ISSUE #210: Add send message endpoint request
https://dev.twitch.tv/docs/api/reference/#send-chat-message

### PR DESCRIPTION
**Pull request by diPhantxm**
Closes #210

### CODE REVIEW

File: chat.go [hunk 1]
```diff
@@ -428,3 +428,78 @@ func (c *Client) UpdateUserChatColor(params *UpdateUserChatColorParams) (*Update
...
+	// The message to send. The message is limited to a maximum of 500 characters.
+	// Chat messages can also include emoticons.
+	// To include emoticons, use the name of the emote.
+	// The names are case sensitive.
+	// Don’t include colons around the name (e.g., :bleedPurple:).
+	// If Twitch recognizes the name, Twitch converts the name to the emote before writing the chat message to the chat room
+	Message string `query:"message"`
+
+	// The ID of the chat message being replied to
+	ReplyParentMessageID string `query:"reply_parent_message_id"`
```
Comment (Satont) [inline chat.go:448]: Thats a POST request, so it should be body fields.

Also `reply_parent_message_id`, so there should be `omitempty` on this field.

File: chat.go [hunk 2]
```diff
@@ -428,3 +428,78 @@ func (c *Client) UpdateUserChatColor(params *UpdateUserChatColorParams) (*Update
...
+// and either channel:bot scope from broadcaster or moderator status
+func (c *Client) SendChatMessage(params *SendChatMessageParams) (*ChatMessageResponse, error) {
+	if params.BroadcasterID == "" {
+		return nil, errors.New("error: broadcaster id must be specified")
+	}
+	if params.SenderID == "" {
+		return nil, errors.New("error: sender id must be specified")
+	}
+
+	resp, err := c.put("/chat/messages", &ManyChatMessages{}, params)
```
Comment (Satont) [inline chat.go:495]: Thats a post, not put.

File: chat.go (hunk 1 above)
Comment (Satont) [reply chat.go:448]: https://dev.twitch.tv/docs/api/reference/#send-chat-message

File: chat.go (hunk 2 above)
Comment (Satont) [reply chat.go:495]: https://dev.twitch.tv/docs/api/reference/#send-chat-message

File: chat.go (hunk 1 above)
Comment (diPhantxm) [reply chat.go]: Sorry for my inattention. Fixed

File: chat.go (hunk 2 above)
Comment (diPhantxm) [reply chat.go]: Fixed

**Review by nicklaw5 — APPROVED** [review]
Thanks @diPhantxm. And thanks for the review @Satont

**Pull request merged**